Documentation/contributor-guide/features.md

```diff
@@ -45,6 +46,38 @@ Any new enhancements to the etcd are typically added as an experimental feature.
     - Add a CHANGELOG entry.
 - At least two maintainers must approve feature requirements and related code changes.
 
-### Graduating an Experimental feature to Stable (TODO - spzala)
+### Graduating an Experimental feature to Stable
 
-### Deprecating a feature (TODO - spzala)
+It is important that experimental features don't get stuck in that stage. They should be revisited and moved to the stable stage following the graduation steps as described here.
+
+#### Locate graduation candidate
+Decide if an experimental feature is ready for graduation to the stable stage.
+- Find an issue that was used to enable the experimental feature initially. One way to find such issues is to search for issues with `type/feature` and `experimental` labels.
```

Comment:
```suggestion
- Find the issue that was used to enable the experimental feature initially. One way to find such issues is to search for issues with `type/feature` and `experimental` labels.
```

Reply:
@ahrtr addressing it in the new update. Thanks!